iotdb-core/node-commons/src/assembly/resources/conf/iotdb-system.properties.template

```diff
@@ -684,6 +684,26 @@ time_partition_interval=604800000
 # Datatype: long
 heartbeat_interval_in_ms=1000
 
+# Default failure detector, enum from {fixed, phi_accrual}
+# effectiveMode: restart
+# Datatype: string
+failure_detector=fixed
+
+# Fixed failure detector threshold of time elapsed without receiving heartbeat replies
+# effectiveMode: restart
+# Datatype: long
+failure_detector_fixed_threshold_in_ms=20000
+
+# Phi accrual: threshold to determine a node is down
+# effectiveMode: restart
+# Datatype: long
+failure_detector_phi_threshold=30
```

Comment:
is it too large?

<img width="1121" alt="image" src="https://github.com/user-attachments/assets/99840f32-7a63-490b-a89a-8b0ef459d1b4" />

Reply:
Actually I think threshold = 30 is a little bit small. From the experiment, we can see that threshold=30 only tolerates about 2-3 seconds of not receiving heartbeats.
<img width="569" alt="image" src="https://github.com/user-attachments/assets/9be1df37-e10e-4d16-bfde-21579be2dde3" />